main.js

```diff
@@ -1 +1,145 @@
-  
+ /*
+ * Created by Jenya on 13.06.15.
+ */
+function onScreen (text) {
+    document.getElementById("p1").innerHTML += "<br>" + text;
+}
+
+var textLoaded="";
+var sourceData=[];
+var casesSizes=[];
+var casesTotals=[];
+var casesOutputs=[];
+
+function loadFileAsText() {
+    var fileToLoad = document.getElementById("fileToLoad").files[0];
+    var fileReader = new FileReader();
+    fileReader.onload = function(fileLoadedEvent) {
+        textLoaded = fileLoadedEvent.target.result;
+        divide(textLoaded);
+        loadCases();
+        searcher();
+    };
+    fileReader.readAsText(fileToLoad, "UTF-8");
+}
+
+function loadLocalFile() {
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.onreadystatechange = function(){
+        if(xmlhttp.status == 200 && xmlhttp.readyState == 4){
```

Comment:
Do not compare with ==, compare with === unless you specifically want type coercion